llvm/include/llvm/TargetParser/RISCVISAInfo.h

```diff
@@ -21,6 +21,10 @@
 
 namespace llvm {
 
+namespace RISCV {
+static constexpr unsigned FeatureBitSize = 2;
```

Comment:
Can we put this in RISCVISAInfo? It describes the maximum value returned by RISCVISAInfo::getRISCVFeaturesBitsInfo.

Reply:
I understand the review comment now. I previously misunderstood it. I now think that FeatureBitSize is a member of `RISCVISAInfo` and can be retrieved using `RISCVISAInfo::FeatureBitSize`.